source/styleEditor.js

```diff
@@ -1,3 +1,9 @@
 $(document).ready(function(){
-  // your code goes here.
+  $( "#style_editor" ).submit(function( event ) {
+    var selector = $("input[name='selector']").val()
+    var property = $("input[name='property']").val()
+    var css_value = $("input[name='value']").val()
+    $(selector).css(property, css_value)
+    event.preventDefault()
```

Comment:
I didn't realize you could put `event.preventDefault()` at the bottom, but hey! I think it's helpful for readability to put it at the top still.